### DISCUSSION TITLE
Understanding the 'celery' queue in /status.json

### DISCUSSION THREAD

**Discussion by 2357gi** [getredash/redash#6925] [Q&A] [answered]
I am currently working on a project where I am using the Redash API to fetch the status of various queues from the /status.json endpoint. I noticed that one of the queues is named 'celery'. 

I am interested in understanding more about this 'celery' queue. Could anyone provide some insights on what this queue is used for and how it operates within the context of Redash? Any information on how items are added to this queue, processed, and removed would also be greatly appreciated. 

Thank you in advance for your help.  

Best Regards, 2357gi

**Comment by justinclift** [comment, marked as answer]
Sure.  The `celery` queue you're asking about is only present in very old versions of Redash.

Those older versions were using [Celery](https://pypi.org/project/celery/) as a job distribution system, to farm out the various tasks to "worker" nodes.

Celery was ok as an initial system, but wasn't really reliable enough (at least back then), so was replaced by [RQ](https://python-rq.org).

It's *probably* a good idea to try and get your Redash system updated to something modern.  That being said, if it's working for you as is and secured properly, then it shouldn't really go wrong.
